src/actions/user.js

```diff
@@ -1,55 +1,47 @@
 const ACTIONS = {
-    LOAD_USER_PROFILE: "LOAD_USER_PROFILE",
-    LOAD_USER_PROFILE_SUCCESS: "LOAD_USER_PROFILE_SUCCESS",
-    LOAD_USER_PROFILE_FAILURE: "LOAD_USER_PROFILE_FAILURE",
-    LOGOUT_USER: "LOGOUT",
-    LOGOUT_USER_SUCCESS: "LOGOUT_SUCCESS",
-    SAVE_PROFILE: "SAVE_PROFILE",
-    SAVE_PROFILE_SUCCESS: "SAVE_PROFILE_SUCCESS",
-    SAVE_PROFILE_FAILURE: "SAVE_PROFILE_FAILURE"
-};
-
-export const USER_ACTIONS = ACTIONS;
+  LOAD_USER_PROFILE: 'LOAD_USER_PROFILE',
+  LOAD_USER_PROFILE_SUCCESS: 'LOAD_USER_PROFILE_SUCCESS',
+  LOAD_USER_PROFILE_FAILURE: 'LOAD_USER_PROFILE_FAILURE',
+  LOGOUT_USER: 'LOGOUT',
+  LOGOUT_USER_SUCCESS: 'LOGOUT_SUCCESS',
+  SAVE_PROFILE: 'SAVE_PROFILE',
+  SAVE_PROFILE_SUCCESS: 'SAVE_PROFILE_SUCCESS',
+  SAVE_PROFILE_FAILURE: 'SAVE_PROFILE_FAILURE',
+}
+
+export const USER_ACTIONS = ACTIONS
 
 export const loadUserProfile = () => ({
-    type: ACTIONS.LOAD_USER_PROFILE
-});
-
-export const loadUserProfileSuccess = (profile) => ({
-    type: ACTIONS.LOAD_USER_PROFILE_SUCCESS,
-    id: profile.id,
-    created: profile.dateCreated,
-    email: profile.email,
-    publicKey: profile.publicKey,
-    displayName: profile.displayName,
-    exercises: profile.exercises
-});
+  type: ACTIONS.LOAD_USER_PROFILE,
+})
+
+export const loadUserProfileSuccess = profile => Object.assign({ type: ACTIONS.LOAD_USER_PROFILE_SUCCESS }, profile)
```

Comment:
To be consistent with the modifications done above
```suggestion
export const loadUserProfileSuccess = profile => ({ type: ACTIONS.LOAD_USER_PROFILE_SUCCESS, ...profile})
```